FEM/rf_mfp_new.cpp

```diff
@@ -596,6 +597,16 @@ std::ios::pos_type CFluidProperties::Read(std::ifstream* mfp_file)
                 if (T_Process || fluid_id == 1)
                     viscosity_pcs_name_vector.push_back(arg2);
             }
+            // JM_visco
+            if (viscosity_model == 10)  // my(C,T),
+            {
+                in >> C_0;
+                in >> viscosity_T_shift;
```

Comment:
This is not need. Temperature is changed to Kevin one during the computation.
There is a keyword in pcs: `$TEMPERATURE_UNIT` for input data. 
The default temperature unit for input is Celsius.

Reply:
Thanks for the comment. I will adapt this

Do I understand correctly that internally the temperature is always considered in Kelvin?

I rebased this changes from a quite old version of ogs5 we used for a project and think we received temperature values in celsius then as input for this function

Reply:
You are right that the internal  internally the temperature is always considered in Kelvin. The Celsius can be used for input and output (see PR.  #115). By default, the temperature unit for input and output is Celsius.

If you have to read a temperature value, e.g a reference temperature, for a material model, you can do like the following
```
               in >> T_0;
                T_0 += TemperatureUnitOffset();
```
(see https://github.com/ufz/ogs5/blob/master/FEM/rf_mfp_new.cpp#L363-L364)

In your case, the input of `in >> viscosity_T_shift` is unnecessary.

Reply:
Thanks for referencing the PR. We hadn't noticed that change. I will adapt the changes accordingly